src/pages/Vault/index.tsx

```diff
@@ -1,8 +1,49 @@
 /** @jsxImportSource @emotion/react */
-import React from 'react';
+import React, { useContext } from 'react';
 
-const VaultUi: React.FC = () => <div>Vault</div>;
+import { Spinner } from 'components';
+import { AuthContext } from 'context/AuthContext';
+import { useGetVaults } from 'clients/api';
+import { Vault } from 'types';
+import VaultItem from './VaultItem';
+import { useStyles } from './styles';
 
-const Vault: React.FC = () => <VaultUi />;
+export interface IVaultUi {
+  vaults: Vault[];
+}
 
-export default Vault;
+export const VaultUi: React.FC<IVaultUi> = ({ vaults }) => {
+  const styles = useStyles();
+
+  if (vaults.length === 0) {
```

Comment:
On mainnet this never return this XVS vault

Reply:
Sorry I don't understand what you mean

Reply:
I expected the XVS vault to be returned when fetching the pools on mainnet. When I pulled down the PR and ran it on the mainnet no vaults are returned and it loads indefinitely.

Reply:
Yeah it should be. I just tried and it worked for me, maybe you need to pull?